index.html

```diff
@@ -103,9 +103,11 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                         <div class="input-group">
                             <input type="text" class="form-control py-3" placeholder="Enter your Email">
                             <div class="input-group-append">
-                                <div class="input-group-text subscribe-button text-white px-5 font-weight-bold">
-                                    Contact Us
-                                </div>
+                                <button class="btn">
+                                    <div class="input-group-text subscribe-button text-white px-5 font-weight-bold">
+                                    <a id="contact_button" href="mailto:algophantom0101@gmail.com">Contact Us</a>
```

Comment:
If we use this href method then the form and the input is useless so instead of keeping the mailto here keep it in the form `action` attribute.